ister_test.py

```diff
@@ -131,6 +139,50 @@ def good_template_string_partitions():
     "HTTPSProxy" : "https://proxy.clear.com"}'
 
 
+def cryptsetup_wrapper(func):
+    """Wrapprt for test whose function use pycryptsetup"""
+    @functools.wraps(func)
+    def wrapper():
+        """CryptSetup Mock Class"""
+        class mock_CryptSetup:
+            def __init__(self, device=None, name=None):
+                if device:
+                    COMMAND_RESULTS.append(device)
+                if name:
+                    COMMAND_RESULTS.append(name)
+
+            def luksFormat(self, cipher=None, cipherMode=None,
+                keysize=None, hashMode=None):
+                if cipher:
+                    COMMAND_RESULTS.append(cipher)
+                if cipherMode:
+                    COMMAND_RESULTS.append(cipherMode)
+                if keysize:
+                    COMMAND_RESULTS.append(keysize)
+                if hashMode:
+                    COMMAND_RESULTS.append(hashMode)
+
+            def addKeyByPassphrase(self, passphrase1, passphrase2):
+                COMMAND_RESULTS.append(
+                    "{0} - {1}".format(passphrase1, passphrase2))
+
+            def activate(self, name='', passphrase=''):
+                pass
```

Comment:
What do you think about adding to the COMMAND_RESULTS something like "activating", "deactivating instead of just pass here?

Reply:
ammm, ok, it is a good idea